modules/data-entry/src/main/frontend/src/dataHomepage/LiveTable.jsx

```diff
@@ -225,12 +230,14 @@ function LiveTable(props) {
   };
 
   let makeActions = (entry, actions, index) => {
-    let content = actions.map((Action, index) => {
+    let content = actions.map((item, index) => {
+      let Action = item.component;
+      let onDone = item.onActionDone;
       return <Action
         key={index}
         entryPath={entry["@path"]}
         entryName={getEntityIdentifier(entry)}
-        onComplete={refresh}
+        onComplete={() => {refresh(); onDone && onDone();}}
```

Comment:
We can make it even shorter:
```suggestion
        onComplete={() => {refresh(); onDone?.();}}
```